client/ionic/src/pages/Triage.tsx

```diff
@@ -22,10 +21,12 @@ class Triage extends React.Component<{}, TriageState> {
 
   render() {
   return (
-    <IonPage className={styles.page}>
+    <IonPage>
       <IonHeader>
         <IonToolbar>
-          <IonTitle>Tab 3</IonTitle>
+          <IonTitle>
+            <IonItem href="/Menu" color="primary">WHO</IonItem>
```

Comment:
does path capitalization matter in Ionic?

Reply:
it didn't appear to in the web app, although that may be just browser behavior.  

good catch.  they should strictly match.

Reply:
fixed